src/main/resources/graphql/flashcards.graphqls

```diff
@@ -0,0 +1,32 @@
+"""
+A flashcard is a set of two or more sides. Each side has a label and a text.
+The label is used to specify which side of the flashcard is being shown to the user first for learning
+and which sides he has to guess.
+"""
+type Flashcard {
+    """
+    Unique identifier of this flashcard.
+    """
+    id: UUID!
+    """
+    List of sides of this flashcard.
+    """
+    sides: [FlashcardSide!]!
+    """
+    The labels of the sides that are shown to the user first (as a question).
+    """
+    questionLabels: [String!]!
```

Comment:
Is there a reason to do it like this instead of saving this information as a separate boolean for each side in FlashcardSide?

Reply:
I had usability in mind. You could just put "Question" here in this array by default, have "Question" also available as a defualt label and save the user the tideous clicking on a checkbox all the time if they just want to create two sided cards.

Reply:
That could also easily be handled frontend-side by having a "default" setup when you create a new flashcard which would automatically create two sides with one side having already checked the checkbox. I think that setup would be cleaner and easier than storing the labels which are questions separately.

Reply:
I don't know if I agree but I'm happy to change it

Reply:
I'm just not a big fan of strings whose only purpose it is to declare that other strings are somehow "special". It's just not a very "explicit" way of declaring intent.